Lib/test/test_turtle.py

```diff
@@ -54,6 +56,24 @@
 """
 
 
+@contextmanager
+def patch_screen():
+    """Patch turtle._Screen for testing without a display.
+
+    We must patch the `_Screen` class itself instead of the `_Screen`
+    instance because instatiating it requires a display.
```

Comment:
```suggestion
    We must patch the _Screen class itself instead of the _Screen
    instance because instatiating it requires a display.
```

Since you didn't use backquotes for `turtle._Screen` I think it's fine to drop it here as well.